omnigibson/robots/manipulation_robot.py

```diff
@@ -141,14 +142,18 @@ def __init__(
                     at least two "fingers" need to touch the object.
                 If "sticky", will magnetize any object touching the gripper's fingers. In this mode, only one finger
                     needs to touch the object.
+            grasping_direction (str): One of {"lower", "upper"}. If "lower", joint lower limit is grasp, 
```

Comment:
maybe make this slightly more clear? e.g.: "lower limit represents a closed grasp"